Components/Utilities/utilities.css

```diff
@@ -0,0 +1,68 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    
+}
+
+
+.h1-size-1{
+    font-size: 2rem;
+    line-height: 1.2;
+}
+
```

Comment:
you can name it as heading-1 instead of h1-size-1, let me know your thought on this